src/components/AddBareMetalHosts/AddBareMetalHosts.tsx

```diff
@@ -29,23 +27,27 @@ const AddBareMetalHosts: React.FC<{ cluster: Cluster }> = ({ cluster }) => {
   };
 
   return (
-    <>
-      <ClusterBreadcrumbs clusterName={cluster.name} />
-      <PageSection variant={PageSectionVariants.light} isMain>
+    <Card>
+      <CardTitle>
+        <Title headingLevel="h2" size="lg" className="card-title">
+          Add Bare Metal Hosts
```

Comment:
```suggestion
          Bare Metal Discovery
```

Changing my previous change

Reply:
done